src/sections/Community/Handbook/repo-data.js

```diff
@@ -99,6 +99,15 @@ export const repo_data = [
         link: ["https://layer5.io/community/members/ekene-leonard-nwobodo"],
         repository: " https://discuss.layer5.io",
       },
+      {
+        project: "Service Mesh Pattern",
+        image: patterns,
+        site: "https://github.com/service-mesh-patterns/service-mesh-patterns",
```

Comment:
Let's use the actual site URL. No harm in that

Reply:
This one -> https://meshery.io/ ?

Reply:
Nope.
https://service-mesh-patterns.github.io/service-mesh-patterns/

Reply:
oh github pages , that was silly on my part :sweat_smile: